INSTALL.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
-cd $HOME
-mkdir .shells
+mkdir $HOME/.shells
```

Comment:
The use of `cd` here was redundant, hence combining.

Reply:
Good Evening,

Thank you for reviewing my files !

I in fact uploaded them here in order to find the same configuration files on my different machines, especially at work.

Thank you
I don't use any Mac nor BSD, and most of these are still work in progress. That's why my INSTALL.sh is still largely incomplete. I will complete and correct the files according to your pull requests.

Again, thanks for your help and time !
Cheers,

Guillaume

Reply:
You are more than welcome, Guillaume. :)